makeRelease.sh

```diff
@@ -14,5 +14,5 @@ rm CONTRIBUTORS
 ./createTranslatorsList.py
 
 ./gradlew clean assemblePlayDebug assemblePlayRelease
-cp app/build/outputs/apk/app-play-debug.apk $DEST/AntennaPod_${VERSION_NAME}_debug.apk
-cp app/build/outputs/apk/app-play-release.apk $DEST/AntennaPod_${VERSION_NAME}_release.apk
+cp app/build/outputs/apk/app-play-debug.apk "$DEST"/AntennaPod_"${VERSION_NAME}"_debug.apk
```

Comment:
Are you sure that this works? Why did you add quotes in the middle of the path?

Reply:
Lint provides [this](https://github.com/koalaman/shellcheck/wiki/SC2086) as the lint explanation. I'll change it back.

Reply:
That link mentions quoting the whole path or full segments, not parts of segments. I think quoting the full path would be a good idea